js/battle.js

```diff
@@ -2107,8 +2107,9 @@ var Battle = (function () {
 		// 5 = seeking
 		this.playbackState = 0;
 
-		this.backdropImage = BattleBackdrops[0];
-		this.backdropImage = BattleBackdrops[Math.floor(Math.random() * BattleBackdrops.length)];
+		this.backdropImage = BattleBackdrops.gen6[0];
```

Comment:
The default isn't used for anything. I'd say remove it so it's clear it's not used for anything.